A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_Init.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * Value is rounded to 1000 and fixed to 1 decimal
+ * @param {number | string} value
+ */
+function formatWeight(value) {
+    return (+value).toFixed(1);
+}
+
+function formatTime(timestamp) {
+    var date = new Date(+timestamp * 1000);
+    return `${date.getUTCHours().toString().padStart(2, "0")}${date.getUTCMinutes().toString().padEnd(2, "0")}`;
+}
+
+class CDUAocInit {
+    static ShowPage(mcdu) {
+        mcdu.clearDisplay();
+        mcdu.page.Current = mcdu.page.AOCInit;
+        mcdu.activeSystem = 'ATSU';
+
+        let fltNbr = '_______[color]amber';
+        let originIcao = '____[color]amber';
+        let destinationIcao = '____[color]amber';
+        let ete = "____[color]amber";
+        let fob = `{small}---.-{end}[color]white`;
+        let requestButton = "INIT DATA REQ*[color]cyan";
+        let gmt = "0000[color]green";
+
+        const seconds = Math.floor(SimVar.GetGlobalVarValue("ZULU TIME", "seconds"));
+        gmt = `{small}${FMCMainDisplay.secondsTohhmm(seconds)}{end}[color]green`;
+
+        function updateView() {
+            if (mcdu.page.Current === mcdu.page.AOCInit) {
+                CDUAocInit.ShowPage(mcdu);
+            }
+        }
+
+        mcdu.refreshPageCallback = () => {
+            CDUAocInit.ShowPage(mcdu);
+        };
+        SimVar.SetSimVarValue("L:FMC_UPDATE_CURRENT_PAGE", "number", 1);
+
+        if (mcdu.simbrief.sendStatus !== "READY" && mcdu.simbrief.sendStatus !== "DONE") {
+            requestButton = "INIT DATA REQ [color]cyan";
+        }
+        if (mcdu.simbrief.originIcao) {
+            originIcao = `${mcdu.simbrief.originIcao}[color]cyan`;
+        }
+        if (mcdu.simbrief.destinationIcao) {
+            destinationIcao = `${mcdu.simbrief.destinationIcao}[color]cyan`;
+        }
+        if (mcdu.simbrief.icao_airline || mcdu.simbrief.flight_number) {
+            fltNbr = `{small}${mcdu.simbrief.icao_airline}${mcdu.simbrief.flight_number}{end}[color]green`;
+        }
+        if (mcdu.simbrief.ete) {
+            ete = `${formatTime(mcdu.simbrief.ete)}[color]cyan`;
+        }
+
+        const currentFob = formatWeight(mcdu.getFOB());
+        if (currentFob) {
+            fob = `{small}${currentFob}{end}[color]green`;
+        }
+
+        const display = [
+            ["INIT/REVIEW", "1", "2", "AOC"],
+            ["{big}FMC FLT NO{end}", "{big}GMT{end}"],
+            [fltNbr, gmt],
+            ["{big}DEP{end}"],
+            [originIcao],
+            ["{big}DEST{end}"],
+            [destinationIcao, "CREW DETAILS>[color]inop"],
+            ["{big}FOB{end}"],
+            ["   " + fob],
+            ["{big}ETE{end}"],
+            [ete, requestButton],
+            ["", "{big}ADVISORY{end} "],
+            ["<AOC MENU"]
+        ];
```

Comment:
Could you provide a RL reference for this? I was under the assumption that the MCDU could only display exactly small and large lines interleaved, with large lines optionally being small. I can't recall having ever seen large characters in a label row.

Reply:
Nevermind, I found one:

![image](https://user-images.githubusercontent.com/929769/102230315-065f1d00-3eed-11eb-9a16-0d4b307ecbdf.png)

I can understand that one, but the way it's used in this code seems very weird... the combination label/input is small/large in every single other instance in the entire FMS that I can think of. Do you have the reference you built this off of?

Reply:
I found this odd as well. But I have these references:
![initpage1](https://user-images.githubusercontent.com/578713/102236696-90e65300-3ed2-11eb-9695-fe9fb9dd64ef.png)
![initpage2](https://user-images.githubusercontent.com/578713/102236706-92b01680-3ed2-11eb-88de-9e7a80f6367e.png)

Reply:
Those are not real references - these are often photoshopped in the original documentation and have inaccurate sizes and layouts. Unless we have some contrary confirmation from a pilot or a phote I would suggest we assume that this is small labels + large content in reality.

Reply:
![fslabs1](https://user-images.githubusercontent.com/578713/102252994-8da89280-3ee5-11eb-9e02-12369c4a66e0.png)
This one is from the FSLabs MCDU. I will ask feedback from pilots on the pilot-feedback channel

Reply:
Sounds good. Otherwise I'm okay with taking FSLabs as confirmation here, they're usually pretty accurate. It's just very unusual.

Reply:
@wpine215 Regarding the {big} fields. Ive followed the docs (it can be inaccurate as pointed out by @beheh). Ive asked for pilots reference on the discord channel, but I still doesn't have an answer

Reply:
I think we should stick with small labels and large content as a final decision, as @beheh suggested, since we don't have any IRL references, and AOC's vary from airline to airline anyway.